vcd/datasource_vcd_nsxt_alb_service_engine_group.go

```diff
@@ -63,6 +62,12 @@ func datasourceVcdAlbServiceEngineGroup() *schema.Resource {
 				Optional:    true,
 				Description: "Boolean value that shows if virtual services are overallocated",
 			},
+			"supported_feature_set": {
+				Type:        schema.TypeString,
+				Optional:    true,
```

Comment:
No need to set for `Optional: true` in the datasource. We never accept this as a parameter.

Reply:
Yeah, you're right. Removed.